packages/block-library/src/navigation-menu-item/editor.scss

```diff
@@ -50,13 +50,36 @@
 }
 
 .wp-block-navigation-menu-item {
-	font-family: inherit;
-	font-size: inherit;
-	background-color: var(--background-color-menu-link);
-	color: var(--color-menu-link);
+	&:not(.is-editing) {
+		font-family: inherit;
+		font-size: inherit;
+		background-color: var(--background-color-menu-link);
```

Comment:
Not a specific problem form this PR, but I think we can not use CSS variables as they are not supported by IE11 https://caniuse.com/#feat=css-variables.
I guess given that it was an existing problem, this is not a blocker at all, but later we should apply some refactoring.

Reply:
The decision for the nav menu block was to implement it using forward looking tech and simply gracefully degrade it for IE, in this case not showing the color controls at all.

Reply:
Actually we've created [a PR that removes the CSS vars](https://github.com/WordPress/gutenberg/pull/18172).